### ISSUE TITLE
Medium Priority, we need a feedback button somewhere, PLEASE!!

### ISSUE THREAD

**Issue by GriffGreen** [GeneralMagicio/pw-frontend#175]
We need a feedback button somewhere, if a badge holder runs into an issue, they should have a way to report it... even a direct link to creating an issue in our github is fine, our audience is 150 people like me... OG & trusted founders, devs, and OP peoplel, 90% of them have made 10 or more github issues I promise.

**Comment by krisjpi** [comment]
We had mentioned this feature a little while ago, and I am happy to have any feedback route directly to me if preferred (Twitter or Discord message).

I also really like the idea of having a pathway to send feedback directly to github issues.

I agree this button is needed with at least medium-high priority

**Comment by kristoferlund** [comment]
@krisjpi @GriffGreen 

A button called "Feedback", leading directly to the github issues? So, this is about collecting feedback on how the website works?

Other feedback, like "This project is in the wrong category!", should we invite that as well?

**Comment by GriffGreen** [comment]
Yeah I would invite all feedback into one very well highlighted button that is always there!

**Comment by kristoferlund** [comment]
> Yeah I would invite all feedback into one very well highlighted button that is always there, on every screen. I think its better to be a github issue personally.

The only section we have that is on every screen is the header. @mosaeedi where would you like to place a feedback link? 

Text, simple: `Feedback`

ps. For now, sending folks to Github is ok. In the longterm we definitely don't want anyone/everyone spamming our issues list with general feedback and ideas for improvements. That should go somewhere else.

**Comment by GriffGreen** [comment]
Did this get added?

**Comment by MoeNick** [comment]
@mosaeedi did not work on the design anymore, if it's safe we start spending some time on it, let us know
@GriffGreen
